mParticle-Apple-SDK/Logger/MPLogger.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  MPLogger.swift
+//  mParticle-Apple-SDK
+//
+//  Created by Brandon Stalnaker on 9/22/24.
+//
+
+import Foundation
+
+public struct MPLogger {
+    
+    public static func MPLogger(loggerLevel: MPILogLevel, format: String, arguments: any CVarArg...) {
```

Comment:
Just a suggestion, but maybe we should mark this private to force the use of the other methods below. There should never be a need to call this one directly

Reply:
that makes sense to me